activesupport/lib/active_support/cache.rb

```diff
@@ -1025,10 +1025,11 @@ def initialize(value, compressed: false, version: nil, expires_in: nil, expires_
         @created_at = 0.0
         @expires_in = expires_at&.to_f || expires_in && (expires_in.to_f + Time.now.to_f)
         @compressed = true if compressed
+        @serialized = false
```

Comment:
We should avoid adding new attributes, people who didn't migrate to the new coder still have Marshaled `Entry` instances in their cache and this would break.